spring-kafka-docs/src/main/antora/modules/ROOT/pages/testing.adoc

```diff
@@ -9,7 +9,7 @@ The `spring-kafka-test` jar contains some useful utilities to assist with testin
 Two implementations are provided:
 
 * `EmbeddedKafkaZKBroker` - legacy implementation which starts an embedded `Zookeeper` instance.
-* `EmbeddedKafkaKraftBroker` - uses Kraft instead of `Zookeeper` in combinder controller and broker modes (since 3.6)
+* `EmbeddedKafkaKraftBroker` - (default) uses Kraft instead of `Zookeeper` in combind controller and broker modes (since 3.1)
```

Comment:
I understand that `combinder` was a typo before, but this `combind` still looks like a typo.
May you mean this:
```suggestion
* `EmbeddedKafkaKraftBroker` - (default) uses Kraft instead of `Zookeeper` in combined controller and broker modes (since 3.1)
```
?